tests/pathway-tests.scm

```diff
@@ -1,19 +1,22 @@
-(define-module (tests pathway-test)
-    #:use-module (srfi srfi-64)
-    #:use-module (opencog)
-    #:use-module (opencog exec)
-    #:use-module (opencog bioscience)
-    #:use-module (annotation gene-go)
-    #:use-module (annotation gene-pathway)
-    #:use-module (annotation biogrid)
-    #:use-module (annotation functions)
-    #:use-module (annotation util)
+#! /usr/bin/env guile
+!#
+
+(use-modules
+	(srfi srfi-64)
+	(opencog)
+	(opencog exec)
+	(opencog bioscience)
+	(annotation gene-go)
+	(annotation gene-pathway)
+	(annotation biogrid)
+	(annotation functions)
+	(annotation util)
 )
 
 
 (test-begin "pathway")
 ;; Load test atomspace
-(primitive-load-path "tests/sample_dataset.scm")
+(primitive-load "../tests/sample_dataset.scm")
```

Comment:
Same as the above comment. Revert the path string should be reverted to its previous version without `..`